kf_lib_data_ingest/common/concept_schema.py

```diff
@@ -133,9 +145,18 @@ class BIOSPECIMEN(PropertyMixin):
         COMPOSITION = None
         EVENT_AGE_DAYS = None
 
+        class COLLECTION_EVENT_AGE(QuantityMixin):
```

Comment:
is this different than the existing "EVENT_AGE"?

Reply:
Hmm..idk..probably not